modules/installation-creating-aws-worker.adoc

```diff
@@ -115,10 +115,22 @@ the CloudFormation template for the security group and roles.
 * `r4.4xlarge`
 * `r4.8xlarge`
 * `r4.16xlarge`
++
+[IMPORTANT]
+====
+If `m4` instance types are not available in your region, such as with
+`eu-west-3`, use `m5` types instead.
+====
 
 . Copy the template from the *CloudFormation template for worker machines*
 section of this topic and save it as a YAML file on your computer. This template
 describes the networking objects and load balancers that your cluster requires.
++
+[IMPORTANT]
+====
+If `m4` instance types are not available in your region, such as with
+`eu-west-3`, you must modify the CloudFormation template use `m5` types.
```

Comment:
If want to use `m5` type, have to update MasterInstanceType.AllowedValues list together to include it in https://docs.openshift.com/container-platform/4.1/installing/installing_aws_user_infra/installing-aws-user-infra.html#installation-cloudformation-control-plane_installing-aws-user-infra